noxfile.py

```diff
@@ -15,10 +14,13 @@
     '4-auth',
     '5-logging',
     '6-pubsub',
-    '7-gce']
+    '7-gce',
+    'optional-kubernetes-engine'
```

Comment:
There are tests in this directory, and I wasn't if they're supposed to be run or not.

Reply:
They hadn't been running, but since they're there, let's try them out.